logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -154,6 +154,11 @@ interface ConversationRepository {
     suspend fun deleteMembersFromEvent(userIDList: List<UserId>, conversationID: ConversationId): Either<CoreFailure, Unit>
     suspend fun observeOneToOneConversationWithOtherUser(otherUserId: UserId): Flow<Either<CoreFailure, Conversation>>
 
+    suspend fun getOneOnOneConversationsWithOtherUser(
```

Comment:
question: why we need to specify 'WithOtherUser', isn't oneOnConversation self-explanatory itself?

Reply:
Maybe we don't I was following the convention set by the already existing method `observeOneToOneConversationWithOtherUser`.